sdk/storage/azure-storage-blobs/vcpkg.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-storage-blobs-cpp",
-  "version-semver": "12.13.0",
+  "version-semver": "9.99.999",
```

Comment:
The version here appears wrong.

Reply:
There is a comment in this PR: https://github.com/Azure/azure-sdk-for-cpp/pull/6445

And there is an email thread to follow this. We can discuss in the email thread and get a conclusion.